src/TapRating.tsx

```diff
@@ -166,7 +173,9 @@ const TapRating: React.FunctionComponent<TapRatingProps> = (props) => {
           {reviews[position - 1]}
         </Text>
       )}
-      <View style={starContainerStyle}>{renderStars(rating_array)}</View>
+      <View testID={`${testID}_starsContainer`} style={starContainerStyle}>
```

Comment:
let's use the name of the style object here so this will be `${testID}_starContainer`